src/components/common/molecules/SearchBar/index.tsx

```diff
@@ -1,14 +1,23 @@
-import { FC } from 'react';
-import { FaSearch } from 'react-icons/fa';
+import { Wrapper, Input, StyledMagnifierIcon } from './styled';
 
-type Props = {};
+type Props = {
+  onSubmit: () => void;
+  placeholderText: string;
+};
 
-const SearchBar: FC<Props> = () => {
+const SearchBar = ({ onSubmit, placeholderText }: Props) => {
   return (
-    <div>
-      <h1>SearchBar</h1>
-      <FaSearch />
-    </div>
+    <Wrapper>
+      <Input
+        onKeyPress={(e: any) => {
```

Comment:
@AllTwos I don't have a problem with small inline functions, but here you even have an if statement, so I would consider moving it upper. Something like `const handleKeyPress = (e) => ...`.
Also, the type of this event is `React.KeyboardEvent<HTMLInputElement>`. Please fix that.

Reply:
Done